### DISCUSSION TITLE
I want to contribute new language

### DISCUSSION THREAD

**Discussion by pcmnn-id** [eritislami/evobot#651] [Q&A]
I want to contribute new language but i don't know how to contribute in github, im new on github btw
i have made Indonesian language by editing the one of the locales file, so anyone can help me to upload my language files?

**Comment by wolf-yuan-6115** [comment]
Use pull request

**Reply by AJB-abinjacob** [reply]
Yes,I can help you I will be in discord 
https://discord.gg/qyTfyQvVMU
